test/utils/SafeCast.test.js

```diff
@@ -43,4 +43,74 @@ describe('SafeCast', async () => {
   }
 
   [8, 16, 32, 64, 128].forEach(bits => testToUint(bits));
+
+  describe('toUint256', () => {
+    const maxInt256 = new BN('2').pow(new BN(255)).subn(1);
+    const minInt256 = new BN('2').pow(new BN(255));
+    const maxUint256 = new BN('2').pow(new BN(256)).subn(1);
+
+    it('casts 0', async function () {
+      expect(await this.safeCast.toUint256(0)).to.be.bignumber.equal('0');
+    });
+
+    it('casts 1', async function () {
+      expect(await this.safeCast.toUint256(1)).to.be.bignumber.equal('1');
+    });
+
+    it(`casts INT256_MAX (${maxInt256})`, async function () {
+      expect(await this.safeCast.toUint256(maxInt256)).to.be.bignumber.equal(maxInt256);
```

Comment:
Can we add a test for this value plus one?

Reply:
Taking into account the two's complement representation, where:
`INT256_MAX` is represented as `2^255 - 1` and is equal to `2^255 - 1`, the maximum positive number that can be represented.
`INT256_MIN` is represented as `2^255` and is equal to `- 2^255`, the minimum negative number that can be represented.

We have that,
`INT256_MAX + 1 = INT256_MIN`, as `2^255 - 1 + 1 = 2^255`, which represents `INT256_MIN`.

For this, we are already testing `INT256_MAX + 1`  (this is, `INT256_MIN` and correctly reverting.

Reply:
You're right of course, I got confused when looking at both the signed and unsigned tests. Thanks!